api/email.py

```diff
@@ -1,5 +1,5 @@
 import smtplib
-from csv import DictWriter
+import csv
```

Comment:
Personally I prefer more controled imports

Reply:
I tend to prefer qualified imports, but apparently there is no way to import a few names while keeping them under a namespace (thanks, Guido). 
Is `from csv import writer as csv_writer` ok? In my opinion `writer` in particular is too loose to be imported unqualified.

Reply:
I'm agree, csv_writer is perfect for me